src/Classes/Controllers/RegisterController.php

```diff
@@ -11,7 +11,7 @@ class RegisterController
     private $renderer;
     private $password;
 
-    function __construct(PhpRenderer $renderer, $password)
+    function __construct(PhpRenderer $renderer, string $password)
```

Comment:
I know you didn't write this but I feel like we need some docblocks...

Reply:
added in 8a65d01